content/guides/_index.md

```diff
@@ -25,13 +25,13 @@ W&B consists of three major components: [Models]({{< relref "/guides/models.md"
 **[W&B Models]({{< relref "/guides/models/" >}})** is a set of lightweight, interoperable tools for machine learning practitioners training and fine-tuning models.
 - [Experiments]({{< relref "/guides/models/track/" >}}): Machine learning experiment tracking
 - [Sweeps]({{< relref "/guides/models/sweeps/" >}}): Hyperparameter tuning and model optimization
-- [Registry]({{< relref "/guides/models/registry/" >}}): Publish and share your ML models and datasets
```

Comment:
Double checking here: Since we are using ref and relrefs and Cloudflare is building, it's safe to assume we are not introducing any new broken links, right?

Reply:
Correct! And this PR also adds redirects.